### PR TITLE
Docs: Add a callout about accessibility on tooltips hiding important content

### PR DESCRIPTION
**Pull request by hannahiss**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_
 
 ### Related issues
 
 Linked to PR #1823 
 
 ### Description
 
 Add some information to developers so that they can use tooltips in the most accessible way.
 
 ### Motivation & Context
 
 During #1823 a11y review, our expert informed us that Boosted tooltips are missing a mechanism is available to dismiss the tooltip without moving pointer hover or keyboard focus. See [WCAG site](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html). 
 
 ### Types of change
 
 
 
 
 - Bug fix (non-breaking which fixes an issue)
 
 ### Live previews
 
 
 
 - <https://deploy-preview-2021--boosted.netlify.app/docs/5.3/components/tooltips/>
 
 ### Checklist
 
 
 
 
 #### Contribution
 
 - [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
 
 #### Accessibility
 
 - [x] My change follows accessibility good practices; I have at least run axe
 
 #### Design
 
 - [x] My change respects the design guidelines defined in [Orange Design System](https://system.design.orange.com/0c1af118d/p/8118d1-web)
 - [x] My change is compatible with responsive display
 
 #### Development
 
 - [x] My change follows the [developer guide](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/wiki/Developer-guide)
 - (NA) I have added JavaScript unit tests to cover my changes
 - (NA) I have added SCSS unit tests to cover my changes
 
 #### Documentation
 
 - [x] My change introduces changes to the documentation and/or I have updated the documentation accordingly
 
 ### Checklist (for Core Team only)
 
 - (NA) My change introduces changes to the migration guide
 - (NA) My new component is added in Storybook
 - (NA) My new component is compatible with RTL
 - (NA) Manually run [BrowserStack tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/browserstack.yml)
 - (NA) Manually test browser compatibility with BrowserStack (Chrome >= 60, Firefox >= 60 (+ ESR), Edge, Safari >= 12, iOS Safari, Chrome & Firefox on Android)
 - [ ] Code review
 - [ ] Design review
 - [ ] A11y review
 
 #### After the merge
 
 - [ ] Manually launch [Percy tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/percy.yml)

### CODE REVIEW

File: site/content/docs/5.3/components/tooltips.md [hunk 1]
```diff
@@ -150,6 +150,10 @@ With an SVG:
   <span class="visually-hidden">Helper</span>
 </button>
 ```
+## Accessibility
```
Comment (louismaximepiton) [inline site/content/docs/5.3/components/tooltips.md:153]: I'd add some blank lines around this title

File: site/content/docs/5.3/components/tooltips.md
Comment (louismaximepiton) [inline site/content/docs/5.3/components/tooltips.md:1]: TBH, I'd put this either at the very top of the page or right after the two info callouts coming from Bootstrap in `## Overview`

File: site/content/docs/5.3/components/tooltips.md [hunk 2]
```diff
@@ -150,6 +150,10 @@ With an SVG:
...
+For accessibility purpose, [content on hover or focus should be dismissible, hoverable and persistent](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html).
+With Boosted tooltips, no mechanism is available to dismiss the tooltip without moving pointer hover or keyboard focus (due to Popper library's limitations).
+So, developers should pay attention that the tooltip never hides an important content.
```
Comment (louismaximepiton) [inline site/content/docs/5.3/components/tooltips.md:156]:
```suggestion
Please pay attention that the tooltip never hides an important content.
```

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Some few things

File: site/content/docs/5.3/components/tooltips.md
Comment (louismaximepiton) [reply site/content/docs/5.3/components/tooltips.md:1]: Yeah OK, let's see what's the pov of Julien!

**Review by louismaximepiton — APPROVED** [review]

File: site/content/docs/5.3/components/tooltips.md
Comment (julien-deramond) [reply site/content/docs/5.3/components/tooltips.md:1]: Maybe you can do it like in the [Dropdowns](https://deploy-preview-2021--boosted.netlify.app/docs/5.3/components/dropdowns/). Overview, then Accessibility, then Examples.

Comment (julien-deramond) [reply site/content/docs/5.3/components/tooltips.md:1]: Linked to my previous comment, there's an issue in our Tooltips docs (https://twbs-bootstrap.netlify.app/docs/5.3/components/tooltips/). **Examples** shouldn't be at the same level as **Overview** (see https://twbs-bootstrap.netlify.app/docs/5.3/components/tooltips/).

So you could probably have:
1. Overview
1. Accessibility
1. Examples

Comment (hannahiss) [reply site/content/docs/5.3/components/tooltips.md:1]: I'd seen it while doing the change...

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**